arduino/builder/sketch.go

```diff
@@ -155,3 +156,60 @@ func MergeSketchSources(sketch *sketch.Sketch) (int, string) {
 
 	return lineOffset, mergedSource
 }
+
+// SketchCopyAdditionalFiles copies the additional files for a sketch to the
+// specified destination directory.
+func SketchCopyAdditionalFiles(sketch *sketch.Sketch, destPath string) error {
+	if err := os.MkdirAll(destPath, os.FileMode(0755)); err != nil {
```

Comment:
at some point we may think to replace those path strings with `paths.Path` objects that have a much cleaner API to handle file operations.